.gitignore

```diff
@@ -1 +1,2 @@
 /target
+.idea
```

Comment:
Can you put this into your personal global gitignore file? See the following blog for instructions for how to set it up:
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/